docs/reference/workflows/monobehaviour/interaction/reader-writers/component-data-updates.md

```diff
@@ -27,7 +26,7 @@ component Health {
 }
 ```
 
-The following examples assume that you have a [GameObject that is linked to a SpatialOS entity]({{urlRoot}}/reference/workflows/monobehaviour/linking/spatialos-entities) containing the `Health` component.
```

Comment:
do we not want to link to the game object creation feature module?

Reply:
I feel like its unnecessary information here

Reply:
it's not necessary information, but if people don't know about the linking, they may want to read more about it

Reply:
_sigh_ 

Fixed